analytics/snippet_tracking.rst

```diff
@@ -0,0 +1,123 @@
+:orphan:
+
+.. highlight:: js
+.. default-domain:: js
+.. _analytics-snippet:
+
+Setting up snippet to track data
+================================
+
+
+Installing tracking code via code snippet.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Installation via snippet should be done only if Tag Manager is not available or you want to track multiple domains / subdomains.
+
+This code should be added near the top of the ``<head>`` tag and before any other script or CSS tags. Additionally
+snippet has to be configured this way:
+
+    * String ``XXX-XXX-XXX-XXX-XXX`` should be replaces with :term:`app ID` (e.g. ``efcd98a5-335b-48b0-ab17-bf43f1c542be``).
+    * String ``ppms.example.com`` should be replaced with your PPMS domain name.
+
+
+.. code-block:: html
+
+    <!-- Piwik -->
+    <script type="text/javascript">
+      var _paq = _paq || [];
+      _paq.push(["trackPageView"]);
+      _paq.push(["enableLinkTracking"]);
+      (function() {
+        var u="//ppms.example.com/";
+        _paq.push(["setTrackerUrl", u+"piwik.php"]);
```

Comment:
help, piwik.js and piwik.php for BC, ppms.php and ppms.js starting from PPMS 5.5.1

Reply:
Do we should write a note below, with info that for older versions piwik is still supported?

Reply:
It's quite around: for older versions only piwik.* is supported, for newers (5.5.1+) we recommend ppms.* but piwik.* will also work as a backward compatibility.

Reply:
Hmm so we could leave it as it is and write a  note about suggesting using ppms when using 5.5.1+

Reply:
Or another way around.

Reply:
I don't know what what's the best practice here. We should probably version the docs. Anyway, it's not like 'name doesn't matter since both ppms.js and piwik.js will work' - we renamed it to ppms.* because it was recognized as Matomo by some software.
We should at least remember to update the doc in near future or add some simple note now.

Reply:
The docs will be versioned. It’s just an initial release. 😄